p2p/peerManager/peerManager.go

```diff
@@ -151,13 +154,18 @@ func NewManager(ctx context.Context, low int, high int, datastore datastore.Data
 		return nil, err
 	}
 
-	return &BasicPeerManager{
+	pm := &BasicPeerManager{
 		ctx:                  ctx,
 		BasicConnMgr:         mgr,
 		BasicConnectionGater: gater,
 		streamManager:        streamManager,
 		peerDBs:              peerDBs,
-	}, nil
+		logger:               log.NewLogger("nodelogs/peers.logs", "debug"),
+	}
+
+	go pm.reportPeerStats()
```

Comment:
need to kill it when manager closes as well

Reply:
Addressed